wandb/util.py

```diff
@@ -215,8 +215,12 @@ def __getattr__(self, key):
 
 
 def app_url(api_url):
-    if "wandb." in api_url and api_url.startswith('https://api.'):
-        return api_url.replace('api.', '')
+    if '://api.wandb.' in api_url:
```

Comment:
why not check for `.ai` too?

Reply:
Basing it on chris' existing code.  I know we have lots of non ai domains out there, i assume he had a reason for excluding the ai check.  staying safe until we can ask

Reply:
I think this check won't work with `api.qa.wandb.ai` -- but that's probably ok because the onprem cloud case below will mostly handle it correctly.